secondary-clusters.md

```diff
@@ -1,11 +1,11 @@
-Kubecost Secondary Clusters
-===========================
+Secondary Clusters Guide
+========================
 
-Secondary clusters are "agent only" clusters that send their metrics to a storage-bucket (aka durable storage) that is accessed by the primary cluster to provide a ___single-pane-of-glass___ view into all aggregated cluster costs globally. This aggregated cluster view is exclusive to `Kubecost Enterprise`.
+Secondary clusters are "agent only" clusters that send their metrics to a central storage-bucket (aka durable storage) that is accessed by the primary cluster to provide a ___single-pane-of-glass___ view into all aggregated cluster costs globally. This aggregated cluster view is exclusive to `Kubecost Enterprise`.
```

Comment:
WHat do we think about changing the "agent-only" verbiage? I am worried this causes confusion between self-hosted and SaaS.

Reply:
The term agent applies, and it is in quotes. I'm happy to add a note about the SaaS solution here though.